packages/core/src/components/map-controls/map-controls.template.ts

```diff
@@ -0,0 +1,48 @@
+import { checkFix, MapControls } from '@dso-toolkit/sources';
+import { html } from 'lit-html';
+
+import { SelectedBaseLayerChangeEvent } from '../map-base-layers/map-base-layers.interfaces';
+import { CheckedOverlaysChangeEvent, Overlay } from '../map-overlays/map-overlays.interfaces';
+
+export function mapControlsTemplate({
+  zoomIn,
+  zoomOut,
+  open,
+  baseLayers,
+  selectedBaseLayer,
+  baseLayerChange,
+  overlays,
+  checkedOverlays,
+  checkedOverlaysChange,
+  disableZoom
+}: MapControls) {
+  return html`
+    <div class="dsosb-map-container-mock" style="background-color: #efefef; height: 500px; position: relative; overflow: hidden">
+      <dso-map-controls
+        @zoomIn=${zoomIn}
+        @zoomOut=${zoomOut}
+        .disableZoom=${disableZoom}
+        ?open=${open}
+      >
+        <form>
```

Comment:
Dit `form` heeft geen submit acties oid.. Hoe zat dat ook alweer, toegankelijk enzo? @gofix

Reply:
https://www.w3.org/TR/UNDERSTANDING-WCAG20/consistent-behavior-unpredictable-change.html

Tja in een javascript applicatie die geen daadwerkelijke 'submit' doet van een 'formulier' zal niet op die traditionele manier een 'change of context' worden getriggered. 

Je zou je zelfs kunnen afvragen of dit wel echt een `<form>` is, hoewel het zeker voordelen biedt als dat wel zo is. Het groeperen van inputs in een formulier zorgt ervoor dat - leunend op standaard browsergedrag - bv. met 'enter' dat form 'gesubmit' kan worden. Ook zullen screenreaders over het algemeen deze groepering herkennen en zodoende de inputs ervan ook gegroepeerd aanbieden.

Reply:
Ik laat het voor nu, vanaf volgende week is het voor de buitenwereld te bekijken en dan vragen we Tim wel om een test. Dank!